### ISSUE TITLE
Source graph firefox extension is blocked by Mozilla

### ISSUE THREAD

**Issue by vinchauhan** [sourcegraph/sourcegraph#4848]
- **Sourcegraph version:** 
- **Platform information:** 

https://bugzilla.mozilla.org/show_bug.cgi?id=1541010

#### Steps to reproduce:

1. https://bugzilla.mozilla.org/show_bug.cgi?id=1541010
2.

**Comment by nicksnyder** [comment]
Hi @vinchauhan,

We have added documentation to explain why our extension was blocked by Mozilla so that you can make an informed decision about whether you wish to keep using our extension: https://docs.sourcegraph.com/integration/firefox_security

This is a duplicate of https://github.com/sourcegraph/sourcegraph/issues/4836
